Xamarin.MacDev/MobileProvisionIndex.cs

```diff
@@ -263,6 +263,24 @@ public void Save (string fileName)
 			}
 		}
 
+		/// <summary>
+		/// Determines the last write time in UTC for the specified directory path or the target directory if the path is a symbolic link.
+		/// Returns the latest date of both.
+		/// </summary>
+		static DateTime GetLastWriteTimeUtcForPath(string path)
```

Comment:
The code formatting doesn't quite match the editorconfig in this repo, please run `dotnet format` in the root directory.